components/Players/PlayerManager.vue

```diff
@@ -388,6 +391,33 @@ export default Vue.extend({
         }
       }
     },
+    addMediaHandlers(): void {
```

Comment:
It would be nice to also set `play` and `pause` here, just to ensure everything is handled the same across browser, and making sure to set `navigator.mediaSession.playbackState`.